redhat-developer/app-services-cli
# feat(consumer-group): add reset-offset command

Implement `rhoas kafka consumer-group reset-offset` command.

Closes #763 

### Verification Steps against mock


1. Start mock server.
```
make mock-api/start
```
2. Select a Kafka instance using
```
rhoas kafka use --id <id-of-instance>
```
3. Run the following command to reset offset for a consumer group:
```
go run ./cmd/rhoas kafka consumer-group reset-offset --id my-app --offset earliest --topic topic-1
```
4. The output should be updated consumers in a tabular form

### Verification Steps against stage (WIP)

1. Create a Kafka instance,a Kafka topic and Kafka consumer group.
2. After consuming a few messages, disconnect the consumer group.
3. Run the following command to reset offset for a consumer group:
```
./rhoas kafka consumer-group reset-offset --id my-group --offset earliest -v
```
4. Run the following command to reset offset for specific consumers in a consumer group:
```
./rhoas kafka consumer-group reset-offset --id my-group --topic rama --offset earliest --partitions "1 2" -v
```

### Type of change


### Checklist
